docs/_guides/codewind-vs-code-quick-guide.md

```diff
@@ -61,28 +61,28 @@ If your organization uses customized application stacks and gives you a URL that
 ### Creating an Appsody project
 Throughout the application lifestyle, Appsody helps you develop containerized applications and maximize containers curated for your usage. If you want more context about Appsody, see the [Appsody welcome page](https://appsody.dev/docs).
 
-1. Under the Explorer pane, select `Codewind`.
-2. Expand `Codewind` by clicking the drop-down arrow.
-3. Hover over the `Projects` entry underneath Codewind in the Explorer pane, and press the `+` icon to create a project.
+1. Under the Explorer pane, select **Codewind**.
+2. Expand **Codewind** by clicking the drop-down arrow.
+3. Hover over the **Projects** entry underneath Codewind in the Explorer pane, and press the `+` icon to create a project.
     * **Note:** Make sure that Docker is running. Otherwise, you get an error.
-4. Choose the `Appsody Open Liberty default template (Appsody Stacks - incubator)`.
-5. Name your project `appsody-calculator`.
-    * If you don't see Appsody templates, find and select `Template Source Manager` and enable `Appsody Stacks - incubator`.
+4. Choose the **Appsody Open Liberty default template (Appsody Stacks - incubator)**.
+5. Name your project **appsody-calculator**.
+    * If you don't see Appsody templates, find and select **Template Source Manager** and enable **Appsody Stacks - incubator**.
     * The templates are refreshed, and the Appsody templates are available.
 6. Press `Enter`.
-    * To monitor your project's progress, right-click your project, and select `Show all logs`. Then, an `Output` tab is displayed where you see your project's build logs.
+    * To monitor your project's progress, right-click your project and select **Show all logs**. Then, an **Output** tab is displayed where you see your project's build logs.
 
 Your project is complete when you see that your application status is running and your build status is successful.
 
 ### Accessing the application endpoint in a browser
 1. Return to your project under the Explorer pane.
-2. Select the Open App icon next to your project's name, or right-click your project and select `Open App`.
+2. Select the Open App icon next to your project's name, or right-click your project and select **Open App**.
 
 Your application is now opened in a browser, showing the welcome to your Appsody microservice page.
 
 ### Adding a REST service to your application
  1. Go to your project's workspace under the Explorer tab.
- 2. Go to `src->main->java->dev->appsody->starter`.
+ 2. Go to `src>main>java>dev>appsody>starter`.
```

Comment:
Maybe separate the folder names like so: `src`>`main`>`java`>`dev`>`appsody`>`starter`